src/main/kotlin/com/zepben/evolve/database/sqlite/cim/tables/iec61970/base/wires/TableGroundingImpedances.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2024 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.database.sqlite.cim.tables.iec61970.base.wires
+
+import com.zepben.evolve.database.sqlite.cim.tables.Column
+import com.zepben.evolve.database.sqlite.cim.tables.Column.Nullable.NULL
+
+@Suppress("PropertyName")
+class TableGroundingImpedances : TableEarthFaultCompensators() {
+
+    val X: Column = Column(++columnIndex, "x", "NUMBER", NULL)
+
+    override val name: String = "grounding_impedance"
```

Comment:
todo: rename to match class. i.e. pluralised.

Reply:
changed